src/Components/Nav/Nav.js

```diff
@@ -2,38 +2,69 @@ import React from 'react';
 import './Nav.scss';
 
 class Nav extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      TitleMenu: [],
+      SpecMenu: [],
+    };
+  }
+
+  componentDidMount() {
+    fetch('http://localhost:3000/data/mock.json', {
+      method: 'GET',
+    })
```

Comment:
```suggestion
    fetch('/data/mock.json')
```
- localhost:3000을 명시하지 않더라도 루트 경로를 명시할 수 있습니다
- get method 디폴트 값입니다
- 어떤 팀원은 폴더명이 data이고 어떤 팀원은 Data인 것 같습니다. git이 대소문자 구별을 못하기 때문에 추후 문제가 될 수 있습니다. 확인 후 다를 경우 통일시켜주세요!

Reply:
수정했습니다!